src/main/java/org/isf/accounting/dto/BillDTO.java

```diff
@@ -0,0 +1,180 @@
+package org.isf.accounting.dto;
+
+import java.util.Date;
+
+import javax.validation.constraints.NotNull;
+
+import org.isf.patient.dto.PatientDTO;
+import org.isf.priceslist.dto.PriceListDTO;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(description = "Class representing a bill")
+public class BillDTO {
+	
+	private Integer id;
+	
+	private Integer patientId; 
+
+	
+	private Integer listId;
+
+	public Integer getPatientId() {
+		return patientId;
+	}
+
+	public void setPatientId(Integer patientId) {
+		this.patientId = patientId;
+	}
+
+	public Integer getListId() {
+		return listId;
+	}
+
+	public void setListId(Integer listId) {
+		this.listId = listId;
+	}
+	@NotNull
+	@ApiModelProperty(notes = "Date of bill creation", example="2020-03-19T14:58:00.000Z", position = 1)
+	private Date date;
+	
+	@NotNull
+	@ApiModelProperty(notes = "Date of bill updated", example="2020-03-19T14:58:00.000Z", position = 2)
+	private Date update;
+	
+	@NotNull
+	@ApiModelProperty(notes = "boolean wich tells if a price list is applied", example="true", position = 3)
+	private boolean isList;
+	
+	@NotNull
+	@ApiModelProperty(notes = "Price list name", example="Basic", position = 4)
+	private String listName;
+	
+	@NotNull
+	@ApiModelProperty(notes = "Is bill belongs to a patient?", example="true", position = 5)
+	private boolean isPatient;
+	
+	@NotNull
+	@ApiModelProperty(notes = "patient name", example="Mario Rossi", position = 6)
+	private String patName;
+	
+	@NotNull
+	@ApiModelProperty(notes = "Bill status", example="O", position = 7)
+	private String status;
+
+	@NotNull
+	@ApiModelProperty(notes = "Bill Amount", example="1000", position = 8)
+	private Double amount;
+	
+	@NotNull
+	@ApiModelProperty(notes = "Bill balance", example="1500", position = 9)
+	private Double balance;
+	
+	@NotNull
+	@ApiModelProperty(notes = "user name who create the bill", example="admin", position = 10)
+	private String user;
+
+	private volatile int hashCode = 0;
```

Comment:
Maybe at this stage we can avoid to use hashCode in DTOs.
We use them just as data transfert object.

Reply:
It's right. I am correcting it.